packages/ffe-tables/less/table.less

```diff
@@ -124,7 +15,7 @@
     }
 
     &__caption {
-        &:extend(.ffe-h4 all);
+        .ffe-h4();
```

Comment:
`.ffe-h4` har [en media query](https://github.com/SpareBank1/designsystem/blob/a5f9cc6efdb12fbae70e8f29e00fa019749b5d93/packages/ffe-core/less/typography.less#L388) som legger til litt mer `margin-bottom` og `line-height` som ikke blir med her når man bruker `.ffe-h4` på denne måte. Er det med vilje?

Reply:
Nei, for så vidt ikke. Endrer det tilbake. Ser verdien i margin-bottom er feil for øvrig, den er hardkodet i stedet for å bruke [variablene](https://design.sparebank1.no/profil/spacing/).